runtime/cmake/caches/mxdptrs.cmake

```diff
@@ -0,0 +1,34 @@
+################################################################################
+# Copyright (c) 2020, 2020 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+################################################################################
+
+set(OMR_GC_POINTER_MODE "mixed" CACHE STRING "")
+
+# OMR_MIXED_REFERENCES_MODE is set to 'static' or 'dynamic' in the extensions CMAKE_ARGS, but a default value is provided here just in case
+set(OMR_MIXED_REFERENCES_MODE "static" CACHE STRING "")
+
+if(OMR_MIXED_REFERENCES_MODE STREQUAL "static")
+	set(OMR_MIXED_REFERENCES_MODE_STATIC ON CACHE INTERNAL "")
+else()
+	set(OMR_MIXED_REFERENCES_MODE_STATIC OFF CACHE INTERNAL "")
+endif()
```

Comment:
This duplicates the setting in https://github.com/eclipse/omr/pull/5657/files, since `OMR_MIXED_REFERENCES_MODE_STATIC` needs to be set early enough that it is available when building `j9vm_gc_glue_full`.

Does this seem reasonable?

Reply:
The duplication isn't necessary as I pointed out privately.